list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code lacks unit tests, which are essential for ensuring the robustness of the application.

Recommendations:
- Implement unit tests using PHPUnit to cover the functionality of listing books. This will help catch bugs early and ensure the code works as expected.
- Example test case:
  ```php
  class BookRepositoryTest extends \PHPUnit\Framework\TestCase {
      public function testListBooks() {
          $connection = $this->createMock(\mysqli::class);
          $repository = new BookRepository($connection);

          $books = $repository->listBooks();

          // Assertions to verify the books were listed...
      }
  }
  ```